### ISSUE TITLE
Adding Correlation Exclusions via API doesnt match UI functionality. 

### ISSUE THREAD

**Issue by cybgit** [MISP/MISP#7755]
### Work environment

| Questions                 | Answers
|---------------------------|--------------------
| Type of issue             | Bug
| OS version (server)       | Ubuntu 18.04
| PHP version               | 7.4
| MISP version / git hash   |  v2.4.143 (15a836b393fd42a610d398cbdf271b3c3d713d6f) 
| Browser                   | Firefox

### Expected behavior
When adding a correlation exclusion via curl / api and including a comment with the value, i expect the value and comment to get written to the table as happens when doing via the UI

### Actual behavior
Comment field passed in json is what is entered as the value of the correlation exclusion and the submitted value is not used/ignored.

### Steps to reproduce the behavior

Using Curl
```
curl --insecure  -H "Authorization: <YOUR API KEY>" \
-H "Content-type: application/json" \
-H "Accept: application/json" \
-X "POST" https://<MISP URL>/correlation_exclusions/add \
-d '{"value": "http://example12.com", "comment": "Example Comment" }'
```

Using PyMISP

![image](https://user-images.githubusercontent.com/23480665/133303421-9dfae912-dd55-4e35-b8d7-3f40fc3d9165.png)

### Logs, screenshots, configuration dump, ...

Screenshot showing UI, curl (bottom left - not working as expected) and browser POST request (working as expected)

![image](https://user-images.githubusercontent.com/23480665/133303327-13363beb-8644-4d62-a58a-4720b00be5cc.png)

**Comment by righel** [comment]
Nice catch @cybgit ,
It's now fixed in `develop` by 3a81f0d2369ee8adf0d398fb1320c33e3e37d7c0.

Thanks.

**Comment by cybgit** [comment]
Awesome. Good work
